src/helpers/schema.sql

```diff
@@ -76,7 +76,7 @@ CREATE TABLE proposals (
 CREATE TABLE votes (
   id VARCHAR(66) NOT NULL,
   ipfs VARCHAR(64) NOT NULL,
-  voter VARCHAR(64) NOT NULL,
+  voter VARCHAR(100) NOT NULL,
   created INT(11) NOT NULL,
   space VARCHAR(64) NOT NULL,
```

Comment:
Like in the other PR, this column is already 100

Reply:
```suggestion
  space VARCHAR(100) NOT NULL,
```